app/src/test/java/mozilla/lockbox/store/SettingStoreTest.kt

```diff
@@ -60,6 +62,20 @@ class SettingStoreTest : DisposingTest() {
         Mockito.verify(editor).apply()
     }
 
+    @Test
+    fun unlockWithFingerprint_newValue() {
+        val newValue = true
+        val defaultValue = false
+
+        unlockWithFingerprint.assertValue(defaultValue)
```

Comment:
I'd like to see a test for the updated `unlockWithFingerprint` value via the `unlockWithFingerprint` observer, and a test around the `FingerprintAuthAction` behavior.